### DISCUSSION TITLE
Leaderboards depends on the language.

### DISCUSSION THREAD

**Discussion by zefr0x** [monkeytypegame/monkeytype#1422] [Ideas]
I don't know any thing about the structure of the database this may not be applicable but I think it's wonderful to have a leaderboard for each language.

**Comment by frogfrogfrogfrogfrog** [comment]
Wouldn't that be quite a bit of leaderboards though? That'd be a lot of data to store. Since you'd have to store the top 1000 global scores and 100 daily scores everyday. Even if you prioritized English and left the other languages to smaller amounts such as the top 100 global and 10 daily I figure that it'd still be clunky to try to find a way to make every language easily accessible.

**Reply by Miodec** [reply]
Also, even if we were to add more language leaderboards, I think only the major languages would be added, to avoid the absolute mess of having 40 leaderboards. Also, I think the other language leaderboards would be smaller than 1000.

**Comment by ShahriarKh** [comment]
Duplicates: #1835, #2059
